pkg/daemon/daemon.go

```diff
@@ -1257,8 +1257,8 @@ func compareOSImageURL(current, desired string) (bool, error) {
 // Otherwise if `false` is returned, then we need to perform an update.
 func (dn *Daemon) checkOS(osImageURL string) (bool, error) {
 	// Nothing to do if we're not on RHCOS
```

Comment:
Let's add "and FCOS"

Reply:
Similar update at https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/daemon.go#L1255